src/index.js

```diff
@@ -1,14 +1,49 @@
 import './stylesheets/style.css';
-import createScores from './create-score.js';
 import createScorePanelHeader from './score-panel.js';
 import createScoreForm from './score-form.js';
+import {
+  parseResponse, fetchScore, refreshScore, createGame,
+} from './utils.js';
+import updateDomWithScores from './dom.js';
 
-window.addEventListener('load', () => {
+window.addEventListener('load', async () => {
   const scores = document.getElementById('scores');
   const header = document.getElementById('score-form-header');
   header.textContent = 'Add your score';
-  const scoresArray = new Array(10).fill(0);
-  createScores(scores, scoresArray);
   createScorePanelHeader();
   createScoreForm();
+  const GAMESNAME = {
+    name: 'wale-leaderboard-microverse',
+  };
+  const { result } = await createGame(GAMESNAME);
+  const gameId = parseResponse(result);
+
+  const addButton = document.getElementById('add-score');
+  const refreshButton = document.getElementById('refresh-score');
+  const user = document.getElementById('your-name');
+  const score = document.getElementById('your-score');
+
+  addButton.addEventListener('click', async (event) => {
+    event.preventDefault();
+    const data = {
+      user: user.value.trim(),
+      score: score.value.trim(),
+    };
+    const postData = {
+      data,
+      gameId,
+    };
+    await fetchScore(postData);
+    user.value = '';
+    score.value = '';
+    const { result } = await refreshScore(gameId);
+    updateDomWithScores(scores, result);
+    user.focus();
+  });
+
+  refreshButton.addEventListener('click', async (event) => {
```

Comment:
Let's double-check on this function, we want the score list to change once the `refresh` button is clicked.

In your case, clicking the `refresh` button doesn't change the score.

Reply:
@olawale-o Apology for the mixed up 🙏 , there was a little glitch at my end, everything seems working fine.

I will be here to make sure this PR is approved.

Reply:
Hi @bellom

I really appreciate your time and effort spent on the review. I believe I understand the reason for the mix up. It is totally fine. 

Please keep up the good work. Thanks

Reply:
What a professional reply 👍 , please keep it up.